### DISCUSSION TITLE
Mouse support?

### DISCUSSION THREAD

**Discussion by pnkpanther65** [complexlogic/flex-launcher#62] [Q&A] [answered]
Sorry if this is the wrong place. I've been setting this up on my htpc and it works great, but is there any way to re-enable the mouse cursor? MouseSelect=true doesn't seem to do what im looking for. I don't always have my controller or keyboard on and tend to use my mouse as a quicker option.  Love how lightweight and customizable this launcher is btw!

**Comment by complexlogic** [comment, marked as answer]
The mouse cursor visibility is [disabled in the code](https://github.com/complexlogic/flex-launcher/blob/master/src/launcher.c#L228), so this is currently not possible. The MouseSelect setting just makes it so you can use a mouse left click to function as the keyboard enter. This is needed to support airmouse devices.

I'm curious what you use case is for having a visble mouse cursor, since mouse navigation (i.e., left and right movement via mouse) is not supported.

**Comment by pnkpanther65** [comment]
Gotcha no worries! I figured that was the case. my use case is an htpc/plex server with a bt kb/mouse setup. i usually use the mouse for quicker navigation in plex and windows anyway (mouse is always on, battery lasts for months, just slightly faster than turning on the keyboard) so haven't opted for a bluetooth remote or anything yet.

thanks for the quick reply!
